src/messages/GE_fr-FR.json

```diff
@@ -319,5 +319,6 @@
   "Administrator permission": "L'administration du locataire doit d'abord obtenir une autorisation avant que vous puissiez donner votre consentement",
   "Adaptive Cards designer": "Concepteur de cartes adaptatives.",
   "Click Run Query button above to fetch Adaptive Card": "Cliquez sur le bouton Exécuter la requête ci-dessus pour aller chercher la carte adaptative.",
-  "Your history includes queries made in the last 30 days": "Votre historique inclut les requêtes effectuées au cours des 30 derniers jours"
+  "Your history includes queries made in the last 30 days": "Votre historique inclut les requêtes effectuées au cours des 30 derniers jours",
+  "Get token details (Powered by jwt.ms)": "Obtenir les détails du jeton (Propulsé par jwt.ms)"
```

Comment:
@sebastienlevert they mean this line. Perhaps you could remove it as there's a team that will handle the translation and all